app/src/test/java/com/jetbrains/handson/mpp/mobile/ApplicationPresenterTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.jetbrains.handson.mpp.mobile
+
+import org.junit.Assert.*
+import org.junit.BeforeClass
+import org.junit.Test
+
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ApplicationPresenterTest {
+
+    companion object {
+        lateinit var presenter: ApplicationPresenter
+
+        @BeforeClass
+        @JvmStatic
+        fun setup() {
+            presenter = ApplicationPresenter()
+        }
+    }
+
+    @Test
+    fun generalPennyConversion_isCorrect() {
+        val pennies = 1357
+        val pounds = presenter.pennyConversion(pennies)
+
+        assertEquals("£13.57", pounds)
+    }
+
+    @Test
+    fun pennyConversion_roundsTo2DP() {
+        var pennies = 1350
+        var pounds = presenter.pennyConversion(pennies)
+
+        assertEquals("£13.50", pounds)
+
+        pennies = 1300
+        pounds = presenter.pennyConversion(pennies)
+
+        assertEquals("£13.00", pounds)
+    }
+
+    @Test
+    fun pennyConversion_oneP() {
```

Comment:
These tests would be more readable if the name was longer and also included the expected outcome.

e.g. `pennConversion_throwsIllegalArgumentException_whenNegativePenniesPassed`